modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/SimpleObjectRelationshipEdge.tsx

```diff
@@ -60,25 +73,33 @@ export function SimpleObjectRelationshipEdge({
 				markerEnd={`url(#${markerStartId})`}
 				style={objectRelationshipEdgeStyle}
 			/>
-
-			<EdgeText
-				label={label}
-				labelBgBorderRadius={4}
-				labelBgPadding={[8, 5]}
-				labelBgStyle={labelBgStyle}
-				labelShowBg
-				labelStyle={labelStyle}
-				onClick={() => {
-					dispatch({
-						payload: {
-							selectedObjectRelationshipId: id,
-						},
-						type: TYPES.SET_SELECTED_OBJECT_RELATIONSHIP_EDGE,
-					});
-				}}
-				x={edgeCenterX}
-				y={edgeCenterY}
-			/>
-		</g>
+			<foreignObject
```

Comment:
This approach was based on this [example](https://v9.reactflow.dev/examples/edge-with-button)